### PR TITLE
Add recording title to recording upload and responses

### PR DESCRIPTION
**Pull request by AustinfHunter**
**Summary:**
Added the title field to the InstructorRecording serializer and handles the title form field for uploads.

**Changes:**
1. Added title to the serialized fields in the InstructorRecording serializer.
2. Added logic to retrieve the title from the form for POST requests to the audio upload endpoint.

**Tests:**
Manually tested the endpoint using swagger. The titles are being sent with the recordings, which can be seen in the following screenshot
title_in_response

All titles are currently empty strings because the recordings were created before adding the title field. 

I was unable to test the upload side since uploading the audio requires additional AWS keys that I do not currently have access to, specifically those required for invoking lambdas. However, I did manually test that the an included title field in a post request would be correctly mapped to the title variable and added to the instantiation of the new InstructorRecording object, so the title _should_ be saved upon successful uploads.

### CODE REVIEW

File: api/views/user_views.py [hunk 1]
```diff
@@ -249,9 +249,9 @@ class UploadAudioView(APIView):
     @transaction.atomic
     def post(self, request, *args, **kwargs):
         instructor = get_object_or_404(Instructor, user=request.user)
-
+        title = request.data["title"]
```
Comment (amhajja) [inline api/views/user_views.py:252]: This line will probably break the they system if we deploy this PR without/before updating the front-end; could you please the necessary changes (one would probably be to use `get`) and make sure that deploying this PR won't break the system?

**Review by amhajja — CHANGES_REQUESTED** [review]
Left a comment.

File: api/views/user_views.py (hunk 1 above)
Comment (AustinfHunter) [reply api/views/user_views.py:252]: Working on the front-end changes now, I'll update this PR when those changes are complete.

File: api/views/user_views.py [hunk 2]
```diff
@@ -249,9 +249,11 @@ class UploadAudioView(APIView):
     @transaction.atomic
     def post(self, request, *args, **kwargs):
         instructor = get_object_or_404(Instructor, user=request.user)
-
+        title = ""
+        if "title" in request.data:
+            title = request.data["title"]
```
Comment (amhajja) [inline api/views/user_views.py:254]: All of this can be replaced with: `title = request.data.get("title", "")`

**Review by amhajja — APPROVED** [review]
I'm approving; feel free to deploy after making the requested change; also, we should merge my hotfix PR so that your deployment contains the bug fix.

File: api/views/user_views.py (hunk 2 above)
Comment (AustinfHunter) [reply api/views/user_views.py:254]: Good to know, fixing that now.

**Pull request merged**